internal/db/sync/manager_test.go

```diff
@@ -1,9 +1,10 @@
 package sync_test
 
 import (
-	"reflect"
 	"testing"
 
+	"gotest.tools/assert"
```

Comment:
Please use `github.com/stretchr/testify/assert`

Reply:
Done -> https://github.com/NethermindEth/juno/pull/365/commits/dc0cddb3b7e629ac3fc1feb2dccb30c63d5aca9d

Reply:
I have some reservations about this. Testify does not allow us to instrument comparisons in `DeepEqual`, which means asserts that try to compare `nil` felts will panic (gotest uses `go-cmp`, which fixes this problem). This has been a [long-standing issue](https://github.com/stretchr/testify/issues/535) in testify that has yet to be fixed. Other projects [have signaled](https://github.com/cosmos/cosmos-sdk/issues/12332) their desire to move away from testify to gotest.

I am personally in favor of exclusively using gotest where necessary, which is what we are already doing in a [few](https://github.com/NethermindEth/juno/blob/674508537411b0ad4a4dc100e7af0705c80963bb/internal/db/transaction/manager_test.go#L15) [places](https://github.com/NethermindEth/juno/blob/674508537411b0ad4a4dc100e7af0705c80963bb/internal/db/block/block_test.go#L58-L68).

Reply:
I am with @joshklop on this, :+1:  on `gotest`

Reply:
I've never needed to use custom compare functions, but I agree with @joshklop on this because we're using it on other tests on the code